matching-notification-api/src/main/java/org/phenotips/matchingnotification/notification/internal/DefaultUserPatientMatchEmail.java

```diff
@@ -142,24 +146,31 @@ protected void setFrom()
     {
         super.setFrom();
         try {
-            Address userEmails = getUserEmails();
+            Set<Address> userEmails = getUserEmails();
             if (userEmails != null) {
-                Address[] address = { userEmails };
-                this.mimeMessage.setReplyTo(address);
+                Address[] address = new Address[userEmails.size()];
+                this.mimeMessage.setReplyTo(userEmails.toArray(address));
             }
         } catch (Exception ex) {
             // do nothing
         }
     }
 
-    private Address getUserEmails()
+    private Set<Address> getUserEmails()
     {
         String userEmail = USERMANAGER.getCurrentUser().getAttribute(USER_PROPERTY_EMAIL).toString();
-        if (StringUtils.isNotBlank(userEmail)) {
-            InternetAddress replyTo = new InternetAddress();
-            replyTo.setAddress(userEmail);
-            return replyTo;
+        Set<Address> emails = new HashSet<>();
+
+        for (String parsedEmail : StringUtils.split(userEmail, ",|;")) {
```

Comment:
Should space be one of the separators? Would anyone list multiple emails like this?
```
email1@domain1.com email2@domain2.com
```

I think it's much more likely to use ` ` than `|` as a separator.